lab.R

```diff
@@ -151,31 +196,64 @@ focal_counties = c('Butte', 'Merced', 'Sacramento', 'Santa Clara')
 # google_mobility
 mob_df = read_csv(file.path('data', 'mobility.csv'))
 
+skimr::skim(mob_df)
+
 #' 1. *How many distinct values of `type` are there?  What does this variable indicate?*  
 #' 
+#' 6
+#' Seems to be a classification of types of travel (residential, business, etc.)
+#' 
 #' 
 #' 
 
 #' 2. *How about `pct_diff`?* 
 #' 
-#' 
+#' It's the difference (in percentage) of travel for a given type for a given day, relative
+#' to the amount of travel performed on the previous day
```

Comment:
Not relative to the previous day.  If that's what they indicated, retail has been in steady exponential decline pretty much across the state since March.  Basically no one would be going to stores.  Take another look at the docs.